torch/fx/passes/runtime_assert.py

```diff
@@ -34,6 +44,161 @@ def _get_example_value(node: fx.Node) -> Optional[str]:
         return None
 
 
+def _get_sym_val(node: fx.Node) -> "Optional[sympy.Expr]":
+    """
+    Returns sympy.Expr from node example value
+    """
+    if (val := _get_example_value(node)) is not None:
+        # dynamo has py_sym_type, export uses sympy.Expr
+        if isinstance(val, py_sym_types):
+            return val.node.expr
+        elif isinstance(val, sympy.Expr):
+            return val
+    return None
```

Comment:
It's not obvious to me why this is needed, we shouldn't ever have sympy.Expr stored in example_value